homework/list.js

```diff
@@ -1,79 +1,83 @@
 'use strict';
+function ToDoList(){
+    this._listNoteContent = [];
+    this._wrapper = document.querySelector('#note-list-wrapper');
+}
+ToDoList.prototype = {
+    start: function(){
+	var promise = this.fetchList();
+	promise.then(
+	    (data) => {
+		this.updateList(data);
+		this.drawList();
+		if (this._listNoteContent.length !== 0){
+		    this.preloadFirstNote();
+		}
+	    });
+	promise.catch(
+	    (reason) => {
+		throw reason;
+	    }
+	);
```

Comment:
Not sure if this is what Promise code should be like.
My understanding is pulling callback out from the function.